src/main/redux/sagas/persist.ts

```diff
@@ -22,20 +22,22 @@ debug("_");
 
 const persistStateToFs = async (nextState: RootState) => {
 
-    // currently saved with pouchDb in one json file.
-    // may be consuming a lot of I/O
-    // rather need to save by chunck of data in many json file
+    const data = {
+        win: nextState.win,
+        publication: nextState.publication,
+        reader: nextState.reader,
+        session: nextState.session,
+    };
 
     debug("start of persist reduxState in disk");
+    await setReduxPersistState(data);
+
+    // TODO : REMOVE THIS !
+    // there are a new redux persistence
```

Comment:
Can you please add "WIP" to the PR title, and make sure the PR is marked as "Draft". Thanks!

Reply:
It is not a WIP PR , but also not a PR to merge now.

Reply:
> It is not a WIP PR

WIP literally means "Work In Progress", and this PR is included in the `v1.6` project with the "In Progress" status ... so this is inconsistent.

> but also not a PR to merge now

In this case, it is definitely a WIP, and should be marked as "Draft".

Reply:
PS: the `TODO : REMOVE THIS !` code comment is what prompted me to remark on the status of the PR (I assume dual DB / FS persistence is not the final logic, right?)